packages/react-components/react-table/library/src/components/TableCell/TableCell.types.ts

```diff
@@ -2,7 +2,7 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 import { TableContextValue } from '../Table/Table.types';
 
 export type TableCellSlots = {
-  root: Slot<'td', 'div'>;
+  root: Slot<'td', 'th' | 'div'>;
```

Comment:
```suggestion
  root: Slot<'td', 'div'>;
```

This shouldn't be necessary - there is `TableHeaderCell` for that

Reply:
The reason for this change is that the DataGridSelectionCell rendered in the header uses the TableCell. Is there another way to implement that?